BasicLayoutsCodelab/app/src/main/java/com/codelab/basiclayouts/MainActivity.kt

```diff
@@ -221,45 +227,52 @@ fun HomeScreen(modifier: Modifier = Modifier) {
     ) {
         Spacer(Modifier.height(16.dp))
         SearchBar(Modifier.padding(horizontal = 16.dp))
+        Spacer(Modifier.height(32.dp))
         HomeSection(title = R.string.align_your_body) {
             AlignYourBodyRow()
         }
+        Spacer(Modifier.height(32.dp))
         HomeSection(title = R.string.favorite_collections) {
             FavoriteCollectionsGrid()
         }
-        Spacer(Modifier.height(16.dp))
     }
 }
 
 // Step: Bottom navigation - Material
 @Composable
 private fun SootheBottomNavigation(modifier: Modifier = Modifier) {
-    BottomNavigation(
-        backgroundColor = MaterialTheme.colors.background,
+    NavigationBar(
+        containerColor = MaterialTheme.colorScheme.surfaceVariant,
         modifier = modifier
     ) {
-        BottomNavigationItem(
+        NavigationBarItem(
             icon = {
                 Icon(
                     imageVector = Icons.Default.Spa,
                     contentDescription = null
                 )
             },
             label = {
-                Text(stringResource(R.string.bottom_navigation_home))
+                Text(
+                    text = stringResource(R.string.bottom_navigation_home),
+                    style = MaterialTheme.typography.labelMedium
```

Comment:
Any reason why we're not using the defaults?

Reply:
removed